{{cookiecutter.project_name}}/.github/next_steps/02_zenodo_integration.md

```diff
@@ -1,5 +1,22 @@
 ---
-title: 'Next step: Zenodo integration'
+title: 'Next step: Enable Zenodo integration'
 ---
 
-Zenodo integration instructions.
+By enabling Zenodo integration, your package will automatically get a DOI which can be used to cite your package. After enabling Zenodo integration for your GitHub repository, Zenodo will create a snapshot and archive each release you make on GitHub. Moreover, Zenodo will create a new DOI for each GitHub release of your code.
+
+To enable Zenodo integration:
+
+1. Go to http://zenodo.org and login with your GitHub account. When you are redirected to GitHub, *Authorize application* to give permission to Zenodo to access your account.
+1. Go to <https://zenodo.org/account/settings/github/> and enable Zenodo integration of your repository by clinking on `On` toggle button.
+1. Your package will get a DOI only after you make a release. Create a new release on <{{cookiecutter.repository}}/releases/new>
```

Comment:
Can we point to `README.dev.md#33-github` instead of `{{cookiecutter.repository}}/releases/new`?

Reply:
Also link is broken as it renders as `</releases/new>`, but will be fixed in #261

Reply:
updated the link